### ISSUE TITLE
Wikipedia [ modifier | modifier le code ] grossly confusing fr-en model

### ISSUE THREAD

**Issue by kpu** [browsermt/students#67]
https://fr.wikipedia.org/wiki/Droupadi_Murmu

![src](https://user-images.githubusercontent.com/247512/181908610-c94dedd7-68d7-4073-b581-65c6d26c4982.png)
![tgt](https://user-images.githubusercontent.com/247512/181908608-a0d4fe75-deb0-4d99-86d5-9c5b47d2b9eb.png)

**Comment by XapaJIaMnu** [comment]
This is an issue with para crawl where often 1 sentence on the source would be aligned to about 20 different sentences on the target side, one of which would be the correct translation, and the rest of them being metadata like the one that you see. We ran Fr paracrawl through the deduplicator on both sides and apparently what happened is that it just remembered the first src trg pair that often turned out to be genuine source sentence aligned to crap metadata on the target side and there you have the result.

**Comment by XapaJIaMnu** [comment]
The problem is so prevalent that dedup threw away 35% of Fr en paracrawl

**Comment by kpu** [comment]
The deduper is designed to take the first input and remove the subsequent ones.  I think you want the one with highest bicleaner score?

**Comment by XapaJIaMnu** [comment]
Yes, but we were in a pinch ;/. Or translate and compute bleu scores with the synthetic translation. Or anything else but what we did.
